### DISCUSSION TITLE
How do I enable the neotest keybindings?

### DISCUSSION THREAD

**Discussion by StephenTurley** [LazyVim/LazyVim#2509] [Q&A] [answered]
So I'm trying to understand how to enable the neotest keybindings. The documentation implies that these should already be set up, but they aren't. See https://www.lazyvim.org/keymaps#neotest

I think I've enabled the adapter correctly for a rust project. My lua/plugins/test.lua has this, but the key maps do not work.

```
return {
  { "rouge8/neotest-rust" },
  {
    "nvim-neotest/neotest",
    opts = { adapters = { "neotest-rust" } },
  },
}
```

I suppose I'd need to just add them to my custom keymap config but it's not clear to me how to execute a lua command within the config. Would appreciate a pointer to some helpful docs or an example. Thank you

**Comment by dpetka2001** [comment, marked as answer]
You need to enable the extra. See [docs](https://www.lazyvim.org/extras/test/core). Easier way (in my opinion) is via `:LazyExtras`.

**Reply by StephenTurley** [reply]
Ok I think that helped, I missed the test.core extra when I did that earlier. I have a new error now but I'll try to figure it out before updating.

**Reply by StephenTurley** [reply]
Thanks that fixed it.
